### DISCUSSION TITLE
2022년을 보내고 2023년을 맞이하며 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#31] [Announcements]
# 2022년을 보내고 2023년을 맞이하며 - 정현수 기술 블로그

2022년 회고와 2023년 계획

https://junghyeonsu.com/posts/2022-retrospect-2023-plan/

**Comment by ch0213** [comment]
🎅🏻 🎄

**Reply by ch0213** [reply]
메리현수마스

**Comment by thinkanddoit** [comment]
준 멋있다 ! 좋은 영향력을 받는 것 같다 ! 새해복 많이 받고 2023년은 더 멋진 준이 되길 바란다 👍🏻

**Reply by junghyeonsu** [reply]
지원이도 격동과 정착의 2023년이 되길

**Comment by leejihoo** [comment]
올해 목표 꼭 이룰수 있기를. 현수님 글보면서 항상 자극 받고 있어요. 앞으로도 좋은 글 기대하겠습니다^^

**Comment by whdgusdl48** [comment]
2023년엔 더 나아갈 현수를 위해
